docs/proposals/docker-registry-2-4.md

```diff
@@ -0,0 +1,57 @@
+# Support for Docker Registry 2.4
+## Abstract
+
+A proposal to update the docker/distribution to v2.4.0.
+
+## Motivation
+
+Primary goal is to migrate to Schema 2 [1].
```

Comment:
It does allow to do the pruning if we know what can be deleted (which we don't without exporting some code). We have a [carry-patch](https://github.com/openshift/origin/pull/5404/commits/aff38976bec1c5cd60a67179d100984a08d52d94) that let's us build a graph of blobs and choose those that are safe to delete. Whit the rebase, there's no need for the patch anymore.